### PR TITLE
Update README.md as an example

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
I am just making an example PR, this will get closed.

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [ ] Ensure that `make build` completes successfully
 - [ ] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [ ] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

**Review by pb-seiso — CHANGES_REQUESTED** [review]
This provides questionable value to the README. Can you describe what you're trying to do here?

**Comment by JonZeolla** [comment]
I think my change provides values can you take another look?

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 easy_infra is a docker container that simplifies and secures Infrastructure as Code deployments.
 
+This is a sample addition
```
Comment (pb-seiso) [inline README.md:5]:
```suggestion
This is a sample addition.
```

**Review by pb-seiso — APPROVED** [review]
Looks good to me!

**Pull request closed without merging**